net/portmapper/portmapper_test.go

```diff
@@ -16,7 +16,7 @@ func TestCreateOrGetMapping(t *testing.T) {
 	if v, _ := strconv.ParseBool(os.Getenv("HIT_NETWORK")); !v {
 		t.Skip("skipping test without HIT_NETWORK=1")
 	}
-	c := NewClient(t.Logf, nil, nil)
+	c := NewClient(t.Logf, nil, nil, nil)
```

Comment:
good ol' nil, nil, nil

Reply:
What's you threshold for when I should switch this to a `Config` struct? 4 params?

Reply:
I kinda like the pattern that constructors only take the required stuff and then the caller can assign to exported fields of whatever the constructor returns. With docs like "you can only set this once before use", etc.

But don't need to do that cleanup now.